src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1624,6 +1624,7 @@ def __init__(self,
         assert service_type
         self.service_type = service_type
         self.service_id = service_id
+        self.name = self.service_name()
```

Comment:
do we need this?

Reply:
I did. since I rebased this, I get the error what RGWSpec doesn't have the attribute name. Whether I use the json file or not. Couldn't see anything I was missing in the class.

Put it here, so every spec will have it.. though I guess I should go check that other Specs fire the error without it.